env/env.js

```diff
@@ -18,6 +18,8 @@ const {
 // These are kept separately to keep development configs out of git
 const { developmentRaw } = process.env.CI ? require('./env.dev.sample.js') : require('./env.dev.nogit.js')
 
+const { statePrimary } = require('./env.statePrimary.js')
```

Comment:
Your call -- do you think it's confusing to have that listed elsewhere?

Reply:
I only did this in case the list of states that needs this type of treatment grows to a considerable size, are we going to add more states to this list in the future? If not I could easily revert this change if needed.

Reply:
It's fine!  Let's keep it this way.